server/cmd/utils/utils.go

```diff
@@ -0,0 +1,18 @@
+package utils
+
+import (
+	"math"
+	"math/rand"
+	"time"
+)
+
+const LENGTH_OF_ROOM_CODE = 4
+
+func GenerateRoomCode() int {
+	rand.Seed(time.Hour.Nanoseconds())
+	roomCode := 0
+	for i := LENGTH_OF_ROOM_CODE - 1; i >= 0; i-- {
```

Comment:
you can do this without a loop

```go
lowerLimit := int(math.Pow10(RoomCodeLength - 1))
upperLimit := int(math.Pow10(4) - math.Pow10(3))
roomCode := lowerLimit + rng.Intn(upperLimit)
```

Reply:
Applied all the changes, re-review please!